docs/platform/howto/disk-autoscaler.md

```diff
@@ -101,14 +109,18 @@ Create an autoscaler endpoint:
     a desired organization and project.
 1.  On the left sidebar, click <ConsoleLabel name="integration endpoints"/>.
 1.  Click **Aiven Autoscaler** > **Add new endpoint**.
-1.  Set the details of the endpoint, and click **Add endpoint**.
+1.  Set the details of the endpoint, and click **Add endpoint**. The **Max. total disk storage (GB)** field includes your plan's storage.
```

Comment:
nitpicking, how about dropping the UI element name (`field`) and using the label only (`Max. total disk storage (GB`)? e.g. `**Max. total disk storage (GB)** shows your plan's storage.`